README.md

```diff
@@ -1283,6 +1283,26 @@ Summary
 <!-- Sections automatically generated by `cargo dev generate-options`. -->
 <!-- Begin auto-generated options sections. -->
 
+#### [`allowed-confusables`](#allowed-confusables)
+
+A list of allowed "confusable" Unicode characters to ignore when enforcing `RUF001`,
+`RUF002`, and `RUF003`.
+
+**Default value**: `[]`
+
+**Type**: `Vec<char>`
+
+**Example usage**:
+
+```toml
+[tool.ruff]
+# Allow minus-sign (U+2212), greek-small-letter-rho (U+03C1), and greek-small-letter-alpha (U+03B1),
```

Comment:
this should say: 

> ... and the asterisk-operator (U+2217)

(assuming you want to use `"*"` and `"∗"` in the example) 😉

Reply:
Thanks!